app/src/main/res/layout-land/fragment_adyen_error.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_adyen_error_view"
+    android:layout_width="match_parent"
+    android:layout_height="160dp"
+    >
+
+  <com.airbnb.lottie.LottieAnimationView
+      android:id="@+id/validation_success_animation"
+      android:layout_width="wrap_content"
+      android:layout_height="0dp"
+      app:layout_constraintBottom_toTopOf="@+id/guideline3"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      app:lottie_autoPlay="true"
+      app:lottie_enableMergePathsForKitKatAndAbove="true"
+      app:lottie_rawRes="@raw/adyen_error_animation"
+      />
+
+  <TextView
+      android:id="@+id/error_message"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:textAlignment="center"
+      android:textColor="@color/pending_payment_text_color"
+      android:textSize="16sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/validation_success_animation"
+      tools:text="@string/purchase_card_error_invalid_details"
+      />
+
+  <TextView
+      android:id="@+id/contact_us"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="32dp"
+      android:text="@string/error_contac_us_body"
+      android:textAlignment="center"
+      android:textColor="@color/grey_8a_alpha"
+      android:textSize="12sp"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@+id/error_message"
+      />
+
+  <LinearLayout
```

Comment:
Instead of creating a linear layout, you can use the following to have the same results:

<img width="552" alt="Screenshot 2020-02-17 at 14 34 34" src="https://user-images.githubusercontent.com/5518858/74662926-c6c38980-5192-11ea-8a6e-3a4ec02f48af.png">